src/schema/objects/metadata.yaml

```
@@ -962,6 +1019,53 @@ EthicsApprovals:
   type: array
   items:
     type: string
+EventIdentifier:
+  name: EventIdentifier
+  display_name: Event Identifier
+  description: |
+    The message sent to the eye-tracker which disambiguates the belonging
+    of the sample to a certain event, condition or group.
+    This can be a unique trial identifier, or in the case of
+    continuous presentation (for example video) the annotated event of interest.
+  type: string
+EyeCameraSettings:
+  name: EyeCameraSettings
+  display_name: Eye Camera Settings
+  description: |
+    A field to store any settings that influence the resolution
+    and quality of the eye imagery.
+    Autowhitebalance? Changes in sharpness?
+  type: object
+  items:
+    type: object
+EyeTrackerDistance:
+  name: EyeTrackerDistance
+  display_name: Eye-Tracker Distance
+  description: |
+    Distance (in meters) between the eye-tracker and the participant eye(s).
+  type: number
+  unit: m
+EyeTrackingMethod:
+  name: EyeTrackingMethod
+  display_name: Eye-Tracking Method
+  description: |
+    Method used to track gaze or pupil position:
+    "P–CR" for video based eye-tracking using pupil and corneal reflection;
+    "DPI" for Dual-Purkinje Imaging system, "SSC" for scleral search coils;
+    "EOG" for electro-oculogram; "Limbus" for trackers estimating limbus borders
+    between the iris and sclera; or other.
+    See details here:
+    [10.3758/s13428-021-01762-8](https://doi.org/10.3758/s13428-021-01762-8)
```

@mszinte @Remi-Gau this paper has recently been retracted (details in the retraction note -- do not miss to read them...). Can you think of new references?

```suggestion
    [10.3758/s13428-021-01762-8](https://doi.org/10.3758/s13428-021-01762-8)
```

---

Any news on what could be a good reference here?